compose/wordpress.md

```diff
@@ -88,15 +93,22 @@ This pulls the needed images, and starts the wordpress and database containers,
     Creating my_wordpress_db_1
     Creating my_wordpress_wordpress_1
 
+> **Note:** WordPress Multisite works only on ports `80` and/or `443`.
+If you get an error message about binding `0.0.0.0` to port `80` or `443`
+(depending on which one you specified), it is likely that the port you
+configured for WordPress is already in use by another service.
+
 ### Bring up WordPress in a web browser
 
-If you're using [Docker Machine](/machine/), then `docker-machine ip MACHINE_VM` gives you the machine address and you can open `http://MACHINE_VM_IP:8000` in a browser.
+If you're using [Docker Machine](/machine/), then `docker-machine ip MACHINE_VM`
+gives you the machine address and you can open `http://MACHINE_VM_IP:8000` in a
```

Comment:
Maybe "run the command <command> to get the machine address, then open that address in your browser" or something.